langchain4j/src/main/java/dev/langchain4j/service/ServiceOutputParser.java

```diff
@@ -181,7 +236,7 @@ private static String simpleTypeName(Type type) {
             case "java.time.LocalDateTime":
                 return "date-time string (2023-12-31T23:59:59)";
             default:
-                return type.getTypeName();
+                return "Object";
```

Comment:
I would still keep the type name, as it can be a very good hint for an LLM in some cases.
Maybe in the parentheses like this: `Object (Person)`?